src/render-html/messageAsHtml.js

```diff
@@ -10,17 +10,27 @@ const messageDiv = (id, msgClass, flags) =>
      ${flags.map(flag => `data-${flag}="true" `).join('')}
     >`;
 
-const briefMessageAsHtml = ({ id, content, flags, timeEdited, isOutbox, reactions, ownEmail }) => `
+const briefMessageAsHtml = ({
+  auth,
```

Comment:
You can preprocess the data passed in a selector, so you don't need to pass `auth`.